src/exec/run-batch-submitter.ts

```diff
@@ -38,9 +38,9 @@ interface RequiredEnvVars {
   // The maximum number of L2 transactions that can ever be in a batch.
   MAX_BATCH_SIZE: 'MAX_BATCH_SIZE'
   // The maximum amount of time (seconds) that we will wait before submitting an under-sized batch.
-  MAX_BATCH_SUBMISSION_TIME: 'MAX_BATCH_SUBMISSION_TIME'
+  MAX_BATCH_SUBMISSION_TIME: 'MAX_BATCH_SUBMISSION_TIME_S'
```

Comment:
```suggestion
  MAX_BATCH_SUBMISSION_TIME_S: 'MAX_BATCH_SUBMISSION_TIME_S'
```

Ahhh ok I see the issue -- we have to change the keys here too.

Reply:
Same goes for all of the below

Reply:
I created a new pull request with new changes. Tried to edit this existing pull request, but could not figure it out.